### PR TITLE
chore: Add convenience script that builds latest versions of the frontends

### PR DESCRIPTION
**Pull request by klochowicz**
Refer in the docs to the convenience script

### CODE REVIEW

**Comment by klochowicz** [comment]
after getting bitten too many times by the changing frontend, I realised that I really needed some simple way to ensure that both frontends are up to date when I'm testing the app. Happy to just keep it in my personal `gitignore`, but I reckon this might be of use to other devs too.

File: build_frontends.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+cd maker-frontend
+yarn install
+yarn build
+cd ..
+cd taker-frontend
+yarn install
+yarn build
+cd ..
+echo "Maker and taker frontends built successfully"
```
Comment (holzeis) [inline build_frontends.sh:15]: nit: This line may lie... 😄 If you really want to print a success message you should look at the result of the commands above 😉

**Review by holzeis — APPROVED** [review]

**Comment by holzeis** [comment]
🤔 How about a version check on the UI and taker daemon? if these don't match we could show a warning on the frontend - similar on how we do it when the daemon versions do not match. But that might be an overkil..

File: build_frontends.sh (hunk 1 above)
Comment (klochowicz) [reply build_frontends.sh:15]: sorry, how can this lie? I'm using `set -e`, which will stop this script on error

https://www.delftstack.com/howto/linux/bash-set-e/

**Comment by klochowicz** [comment]
> 🤔 How about a version check on the UI and taker daemon? if these don't match we could show a warning on the frontend - similar on how we do it when the daemon versions do not match. But that might be an overkil..

I was toying with augmenting `build.rs` for taker and maker, and recompiling the frontend then. Unfortunately `yarn build` takes 10s even if nothing changes, so that's just more annoying than it's worth.... :|

File: build_frontends.sh (hunk 1 above)
Comment (holzeis) [reply build_frontends.sh:15]: Ups, I've overlooked that..  🙈

**Pull request merged**